<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>4K YouTube to MP3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/image.qrc">
    <normaloff>:/image/application-icon</normaloff>:/image/application-icon</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="HoverListView" name="listView" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>525</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="dragEnabled" stdset="0">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="greetingsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>389</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="greetingsImageLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resource/image.qrc">:/image/paste-url</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="greetingsLabel">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;&quot;&gt;Copy video link from your browser address&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot;&quot;&gt;bar and then click &amp;quot;&lt;/span&gt;&lt;a href=&quot;paste&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Paste URL&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot;&quot;&gt;&amp;quot; here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="linksWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>8</number>
          </property>
          <property name="verticalSpacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLabel" name="linksTitleLabel">
            <property name="text">
             <string>The most popular video hosting services supported by audio downloader:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="youtubeLabel">
            <property name="text">
             <string>&lt;table&gt;
  &lt;tr&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;img src=&quot;:/image/service-youtube&quot; /&gt;&lt;/td&gt;
    &lt;td width=&quot;4&quot;&gt;&lt;/td&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;a href=&quot;http://www.youtube.com/&quot;&gt;www.youtube.com&lt;/a&gt;&lt;/td&gt;
  &lt;/tr&gt;
&lt;/table&gt;
</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="facebookLabel">
            <property name="text">
             <string>&lt;table&gt;
  &lt;tr&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;img src=&quot;:/image/service-facebook&quot; /&gt;&lt;/td&gt;
    &lt;td width=&quot;4&quot;&gt;&lt;/td&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;a href=&quot;http://www.facebook.com/&quot;&gt;www.facebook.com&lt;/a&gt;&lt;/td&gt;
  &lt;/tr&gt;
&lt;/table&gt;
</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="vimeoLabel">
            <property name="text">
             <string>&lt;table&gt;
  &lt;tr&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;img src=&quot;:/image/service-vimeo&quot; /&gt;&lt;/td&gt;
    &lt;td width=&quot;4&quot;&gt;&lt;/td&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;a href=&quot;http://www.vimeo.com/&quot;&gt;www.vimeo.com&lt;/a&gt;&lt;/td&gt;
  &lt;/tr&gt;
&lt;/table&gt;
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="dailymotionLabel">
            <property name="text">
             <string>&lt;table&gt;
  &lt;tr&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;img src=&quot;:/image/service-dailymotion&quot; /&gt;&lt;/td&gt;
    &lt;td width=&quot;4&quot;&gt;&lt;/td&gt;
    &lt;td valign=&quot;middle&quot;&gt;&lt;a href=&quot;http://www.dailymotion.com/&quot;&gt;www.dailymotion.com&lt;/a&gt;&lt;/td&gt;
  &lt;/tr&gt;
&lt;/table&gt;
</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>4</width>
              <height>4</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ExtendedButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource/image.qrc">
      <normaloff>:/image/item-remove</normaloff>:/image/item-remove</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="normalOpacity" stdset="0">
     <number>10</number>
    </property>
    <property name="hoveredOpacity" stdset="0">
     <number>50</number>
    </property>
    <property name="pressedOpacity" stdset="0">
     <number>75</number>
    </property>
   </widget>
   <widget class="ExtendedButton" name="btnAction">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>565</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>action</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="horizontalPadding" stdset="0">
     <number>0</number>
    </property>
    <property name="verticalPadding" stdset="0">
     <number>0</number>
    </property>
    <property name="normalOpacity" stdset="0">
     <number>10</number>
    </property>
    <property name="hoveredOpacity" stdset="0">
     <number>50</number>
    </property>
    <property name="pressedOpacity" stdset="0">
     <number>75</number>
    </property>
   </widget>
   <widget class="ExtendedButton" name="btnTogglePlay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource/image.qrc">
      <normaloff>:/image/item-player-play</normaloff>:/image/item-player-play</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="normalOpacity" stdset="0">
     <number>10</number>
    </property>
    <property name="hoveredOpacity" stdset="0">
     <number>50</number>
    </property>
    <property name="pressedOpacity" stdset="0">
     <number>75</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HoverListView</class>
   <extends>QWidget</extends>
   <header>hoverlistview.h</header>
  </customwidget>
  <customwidget>
   <class>ExtendedButton</class>
   <extends>QPushButton</extends>
   <header>extendedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/image.qrc"/>
 </resources>
 <connections/>
</ui>
